pkg/cli/subcommands/check/command.go

```diff
@@ -141,9 +143,23 @@ func MakeCommand(globalParamsGetter func() GlobalParams) *cobra.Command {
 					LogParams:            logimpl.ForOneShot(globalParams.LoggerName, "off", true)}),
 				core.Bundle(),
 
+				// workloadmeta setup
+				collectors.GetCatalog(),
```

Comment:
dummy question: why this function call is not part of the fx.Provide(...) function?

Reply:
This function calls a collection of fx.provide and converts the collection of fx.options into a single option which is the same as single fx.provide returns.